pkg/cmd/community.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/stripe/stripe-cli/pkg/open"
+)
+
+const communityURL = "https://stripe.com/go/developer-chat"
+
+var openBrowser = open.Browser
+var canOpenBrowser = open.CanOpenBrowser
+
+type communityCmd struct {
+	cmd *cobra.Command
+}
+
+func newCommunityCmd() *communityCmd {
+	cc := &communityCmd{}
+
+	cc.cmd = &cobra.Command{
+		Use:     "community",
```

Comment:
Do we want to call this `community` or `chat`? You could add an alias using `Aliases: []string{"discord", ...}`

Reply:
I felt that `community` was probably the most descriptive, but I've now added aliases for `discord` and `chat`.

Reply:
Sounds good! Do the DA folks have feelings on this? Could imagine a future where `community` might want to go to the community experts or something.